src/vm/gdbjit.cpp

```diff
@@ -2576,11 +2576,25 @@ void NotifyGdb::OnMethodPrepared(MethodDesc* methodDescPtr)
     }
 #endif
 
+    // remove '.ni.dll' or '.ni.exe' suffix from wszModuleFile
+    LPCWSTR pNIExt = const_cast<LPWSTR>(wcsstr(wszModuleFile, W(".ni.exe"))); // where '.ni.exe' start at 
```

Comment:
The const_cast doesn't seem to make sense here. You have created the pNIExt as LPCWSTR (that means const WCHAR *), which is the type that the wcsstr returns. So the pNIExt type should rather be LPWSTR or there should be no cast here and the cast should rather be at the wcscpy's first parameter, since wcscpy expects LPWSTR (that means WCHAR *) as the first argument.

Reply:
Sorry for mistake. I have updated PR.